plugins/netskope/help.md

```diff
@@ -0,0 +1,577 @@
+|data|data|True|Data containing URLs and type|
+|id|integer|True|URL list identifier|
+|modify_by|string|True|Name of the URL list modifier|
+|modify_time|string|True|Last URL list modification time|
+|modify_type|string|True|Shows last modification type. Expected values are Created, Edited, Deleted|
+|name|string|True|URL list name|
+|pending|integer|True|URL list status of pending (1) and applied (0)|
+
+Example output:
+
+```
+{
+	"id": 1,
+	"name": "ExampleName",
+  "data": {
+    "urls": [
+      "https://example.com",
+      "https://example.com"
+    ],
+    "type": "exact",
+    "json_version": 2
+  },
+	"modify_by": "Netskope REST API",
+	"modify_time": "1997-01-01 00:00:00",
+  "modify_type": "Edited",
+  "pending": 0
+}
+```
+
+#### Get All URL Lists
+
+This action is used to get all applied or pending URL lists.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|status|string|None|False|Status of URL lists to be received|['', 'applied', 'pending']|applied|
+
+Example input:
+
+```
+{
+  "status": "applied"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|urllists|[]urllist|True|Contains list of URL list objects|
+
+Example output:
+
+```
+{
+  "urllists": [
+    {
+      "id": 1,
+      "name": "ExampleName",
+      "data": {
+        "urls": [
+          "https://example.com",
+          "https://example.com"
+        ],
+        "type": "exact",
+        "json_version": 2
+      },
+      "modify_by": "Netskope REST API",
+      "modify_time": "2022-01-03T00:00:00.000Z",
+      "modify_type": "Created",
+      "pending": 0
+    }
+  ]
+}
+```
+
+#### Delete a URL List by ID
+
+This action is used to mark a URL list as pending deletion.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|integer|None|True|ID of the URL list|None|1|
+
+Example input:
+
+```
+{
+  "id": 1
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|data|data|True|Data containing URLs and type|
+|id|integer|True|URL list identifier|
+|modify_by|string|True|Name of the URL list modifier|
+|modify_time|string|True|Last URL list modification time|
+|modify_type|string|True|Shows last modification type. Expected values are Created, Edited, Deleted|
+|name|string|True|URL list name|
+|pending|integer|True|URL list status of pending (1) and applied (0)|
+
+Example output:
+
+```
+{
+  "id": 1,
+  "name": "ExampleName",
+  "data": {
+    "urls": [
+      "https://example.com",
+      "https://example.com"
+    ],
+    "type": "exact",
+    "json_version": 2
+  },
+  "modify_by": "Netskope REST API",
+  "modify_time": "1997-01-01 00:00:00",
+  "modify_type": "Deleted",
+  "pending": 0
+}
+```
+
+#### Apply Pending URL List Changes
+
+This action applies currently pending changes for URL lists.
```

Comment:
Can you pick a specific url list to apply the changes? How often this action is needed.

Reply:
We can't pick a specific url list to apply the changes. It applies all the pending ones. This action is needed after making some changes to URL lists such as replacing, updateing, deleting existing ones.